src/ui/server/helpers/summary-lists/export-contract/agent-fields/agent-charges/index.ts

```diff
@@ -42,6 +43,17 @@ const agentChargesFields = (answers: ApplicationExportContractAgentService, refe
 
   if (answers[IS_CHARGING]) {
     if (answers.charge[FIXED_SUM_AMOUNT]) {
+      /**
+       * transforms the number to a whole number if it has .00 at the end
+       */
+      const answer = transformEmptyDecimalsToWholeNumber(answers.charge[FIXED_SUM_AMOUNT]);
+
+      /**
+       * If the number has decimal places, show 2 decimal places
+       * or 0 if it's a whole number
+       */
+      const decimalPlaces = doesNumberHaveDecimalPlaces(answer) ? 2 : 0;
```

Comment:
Not a blocker, could have a helper here for both functions/variables here, so the same logic could be used in other places